datamodel/samples.d/facilities.csv

```diff
@@ -1,852 +1,1026 @@
-NAM HAI PORT,VNHPH,NAMHAI
-TAN VU PORT,VNHPH,TANVU
-SOUTH VAN PHONG INTERNATIONAL PORT,VNNIH,SVP
-CAT LAI TERMINAL,VNSGN,CATLAI
-SAIGON PREMIER CONTAINER TERMINAL (SPCT),VNSGN,SPCT
-CAI MEP INTERNATIONAL TERMINAL (CMIT),VNVUT,CMIT
-SP-SSA INTERNATIONAL TERMINAL (SSIT),VNVUT,VNSST
-TAN CANG - CAI MEP INTERNATIONAL TERMINAL (TCIT),VNVUT,TCIT
-PORT OF APIA,WSAPW,POA
-MULTI PURPOSE TERMINAL (MPT),ZACPT,TNCM
-TRANSNET PORT TERMINALS CAPE TOWN,ZACPT,TNCT
-DURBAN RORO TERMINAL,ZADUR,TNDRT
-GRINDROD MAYDON WHARF TERMINALS,ZADUR,GMWT
-TRANSNET PORT TERMINALS DURBAN PIER 1,ZADUR,TNP1
-TRANSNET PORT TERMINALS DURBAN PIER 2,ZADUR,TNP2
-TRANSNET PORT TERMINALS NGQURA,ZANGQ,TNNQ
-TRANSNET PORT TERMINALS PORT ELIZABETH,ZAPLZ,TNPE
+Facility Name,UNLOCODE,Facility SMDG Code,Facility BIC Code
+EPOLOG EXPORTVERPACKUNG UND LOGISTIK GMBH,DEHAM,,DEHAMAKNM
+Baltic Logistics Service GmbH,DEHAM,,DEHAMALNB
+PROGECO SCHLUISGROVE,DEHAM,,DEHAMBJTV
+Braun Container Handels-GmbH,DEHAM,,DEHAMBRAA
+Hccr,DEHAM,,DEHAMBUSA
+CPS CONPAC PORT SERVICE GMBH,DEHAM,,DEHAMCATK
+Progeco Hamburg Deutschland Gmbh,DEHAM,,DEHAMCDHA
+Cdr container Depot,DEHAM,,DEHAMCDUA
+CMR Hamburg,DEHAM,,DEHAMCMRA
+CST Hamburg,DEHAM,,DEHAMCTHA
+Dcp Dettmer Container Packing,DEHAM,,DEHAMDCBA
+SCHWARZE & CONSORT. GMBH,DEHAM,,DEHAMEGBO
+Eurogate Container Terminal Hamburg (CTH),DEHAM,,DEHAMEGHA
+OCEANGATE DISTRIBUTION GMBH,DEHAM,,DEHAMEROD
+EUROKOMBI Terminal GmbH,DEHAM,,DEHAMFXHE
+Hhla Unikai Terminal,DEHAM,,DEHAMHCCC
+Container Terminal Tollerort (CTT),DEHAM,,DEHAMHCCF
+HCS Hamburger Container Service,DEHAM,,DEHAMHCSA
+Container Terminal Burchardkai (CTB),DEHAM,,DEHAMHHLA
+CONRO CONTAINER GMBH,DEHAM,,DEHAMHLZX
+Hanse-Repair,DEHAM,,DEHAMHNSA
+Hagendorf Planen Service,DEHAM,,DEHAMHPSA
+NEFAB PACKAGING GERMANY GMBH,DEHAM,,DEHAMIJWC
+Universal Container Systems,DEHAM,,DEHAMJABG
+SWT STEINWEG SUED-WEST-TERMINAL,DEHAM,,DEHAMJHUP
+GLOBAL PACKING CENTER GMBH,DEHAM,,DEHAMKJKS
+MEDREPAIR HAMBURG,DEHAM,,DEHAMMNXR
+CONTAINERPRUEFANLAGE (CPA),DEHAM,,DEHAMMPAQ
+PACKING CENTER HAMBURG,DEHAM,,DEHAMNSHE
+TRANSBALTIC UMSCHLAG & SPEDITIONS GMBH,DEHAM,,DEHAMNVUD
+HAFEN LAGER SERVICE GMBH,DEHAM,,DEHAMONXW
+LOGISTIKZENTRUM HAMBURG GMBH,DEHAM,,DEHAMPKUG
+Ernst Logistik Depot,DEHAM,,DEHAMPPWX
+REMAIN GMBH,DEHAM,,DEHAMRMAA
+Remain Container-Depot Hamburg,DEHAM,,DEHAMRMTD
+MEDREPAIR HAMBURG,DEHAM,,DEHAMROSL
+SWOP SEAWORTHY PACKING GMBH,DEHAM,,DEHAMSMCU
+Rexin Container Gruppe,DEHAM,,DEHAMSRXA
+Cotac Depot Hamburg,DEHAM,,DEHAMUGIF
+MT Container GmbH,DEHAM,,DEHAMUVHP
+HLD HANSA LOGISTIK UND DISTRIBUTION GMBH,DEHAM,,DEHAMUVQI
+LOGOO LOGISTIK GMBH NIEDERLASSUNG HAMBURG,DEHAM,,DEHAMVEDT
+SPEDITION KRAUSE,DEHAM,,DEHAMVFKY
+Bimicon Container-Service GmbH,DEHAM,,DEHAMVOJM
+UNITAINER Trading GmbH,DEHAM,,DEHAMVWWG
+HCCR HAMBURGER CONTAINER-/CHASSIS- REPARATUR GMBH.,DEHAM,,DEHAMWDYM
+SCHWARZE & CO,DEHAM,,DEHAMWPAN
+TCO TRANSCARGO GMBH,DEHAM,,DEHAMWTJH
+KROHN & SCHRODER GMBH,DEHAM,,DEHAMWUXF
+Rhenus Midgard Hamburg GmbH,DEHAM,,DEHAMXKPB
```

Comment:
Checking some random values - and this one also exists as a SMDG code here:
RHENUS MIDGARD HAMBURG,DEHAM,RMH,
The BIC codes added here should be added to the already existing SMDG codes where possible

Reply:
Hi @HenrikHL understood and agree this would be great, however the lists are independently managed by BIC and SMDG and published in this way.  

I could group these up when we merge them in the update facilities script to illustrate what you suggest within this `facilities.csv`, but in this case it wouldn't apply as GMBH is missing from SMDG.

There is currently no mapping between the two lists, however we are working through some geofencing topics which may help identify they relate to same facility and this will provide an opportunity to use the datasets to identify this link.  

As these files are examples and the main use in real cases would likely be to pull from API could I suggest doing the grouping to see if we have some examples that have both and show the overlap for this PR?

Reply:
Come to think of it - it might be better to add it the way you have added it now. Reason for this is if both values are present - then SMDG code will be returned.

Reply:
Please keep the list as you have provided it for now :-)